website/docs/guides/version-4-upgrade.html.md

```diff
@@ -433,6 +434,10 @@ output "elasticache_global_replication_group_version_result" {
 
 !> **WARNING:** This topic is placeholder documentation.
 
+## Resource: aws_fsx_ontap_storage_virtual_machine
+
+We removed the misspelled argument `active_directory_configuration.0.self_managed_active_directory_configuration.0.organizational_unit_distinguidshed_name` that was previously deprecated. Use `active_directory_configuration.0.self_managed_active_directory_configuration.0.organizational_unit_distinguished_name` now instead. This is available for state upgrade.
```

Comment:
Maybe instead of "available for state upgrade" which sounds like something left on the practitioner, we could note instead
```suggestion
We removed the misspelled argument `active_directory_configuration.0.self_managed_active_directory_configuration.0.organizational_unit_distinguidshed_name` that was previously deprecated. Use `active_directory_configuration.0.self_managed_active_directory_configuration.0.organizational_unit_distinguished_name` now instead. Terraform will automatically migrate the state from the deprecated attribute. 
```